src/it/scala/org/constellation/ClusterHealthCheckTest.scala

```diff
@@ -64,7 +64,7 @@ class ClusterHealthCheckTest
     runPeriodically(
       apis.foreach { a =>
         val id = a.baseURI
-        val currentMetricCount = a.metrics.getOrElse(metricKey, "0").toInt
+        val currentMetricCount = Await.result(a.metricsAsync, 5 seconds).getOrElse(metricKey, "0").toInt
```

Comment:
What's the difference between `a.metrics` and `Await.result(a.metricsAsync, 5.seconds)`? Each time it will block the thread

Reply:
There is no.

Reply:
So changing it to async, why you've changed it to Future, not IO?

Reply:
bump